release-please-config.json

```
@@ -0,0 +1,11 @@
+{
+  "release-type": "java-yoshi",
+  "separate-pull-requests": false,
+  "include-component-in-tag": false,
+  "group-pull-request-title-pattern": "chore(${branch}): release ${version}",
+  "packages": {
+    ".": {
+      "extraFiles": ["app-gradle-plugin/build.gradle.kts"]
```

[extra-files](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files) 
```suggestion
      "extra-files": ["app-gradle-plugin/build.gradle.kts"]
```

---

done